app/gui/package.json

```diff
@@ -21,7 +21,7 @@
     "build": "vite build",
     "build-cloud": "cross-env CLOUD_BUILD=true corepack pnpm run build",
     "preview": "vite preview",
-    "lint": "eslint . --max-warnings=0",
```

Comment:
Why this change? I want to fail on warnings.

Reply:
yeah, at the very least we should keep it for `project-view`

maybe for the time being we can split the lint command (back) into separate gui and dashboard lint commands filtered by top-level directory?

Reply:
I want to keep distinction between error requiring my immediate attention (because otherwise the program won't  work) from warnings (this is bad, but it may wait before merge). It just improves the development experience, when missing jsdocs won't make the function look like it was completely broken.

To keep everyone happy, maybe we could make the separate script which run linter with additional --plugin and --rule option, which will not be run on CI?

Reply:
From call: we agreed that we keep warnings for react-compiler, but set here limit to their exact current number (so any new warning will still fail the CI) with a comment describing what warnings we want to leave.